### PR TITLE
Revert #395 and implement with alter method

### PR DESCRIPTION
**Pull request by CanglongCl**

### CODE REVIEW

File: Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift [hunk 1]
```diff
@@ -12,6 +12,12 @@ import Magnet
 extension Shortcut {
     // set defalut for app shortcut
     func setDefaultForAppShortcut() {
+        Defaults[.selectionShortcut] = KeyCombo(key: .d, cocoaModifiers: .option)
```
Comment (AkaShark) [inline Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift:15]: Creating a new function named `setDefaultForGlobalShortcut` will be more clearly

**Review by AkaShark — CHANGES_REQUESTED** [review]

**Comment by tisfeng** [comment]
Testing is fine.

**Comment by Jerry23011** [comment]
So far everything works for me, even the release version (2.6.0) works correctly after I cleaned the shortcut

**Comment by phlpsong** [comment]
> So far everything works for me, even the release version (2.6.0) works correctly after I cleaned the shortcut

I have the same issue in #399 with release 2.6.0. This change works well from my side.

File: Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift (hunk 1 above)
Comment (CanglongCl) [reply Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift:20]: Resolved with 27bca4f

File: Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        setDefaultForGlobalShortcut()
+        setDefaultForAppShortcut()
+    }
+
+    private func setDefaultForGlobalShortcut() {
+        Defaults[.selectionShortcut] = KeyCombo(key: .d, cocoaModifiers: .option)
+        Defaults[.snipShortcut] = KeyCombo(key: .s, cocoaModifiers: .option)
+        Defaults[.inputShortcut] = KeyCombo(key: .a, cocoaModifiers: .option)
+        Defaults[.screenshotOCRShortcut] = KeyCombo(key: .f, cocoaModifiers: [.option, .shift])
+        Defaults[.showMiniWindowShortcut] = KeyCombo(key: .s, cocoaModifiers: .option)
```
Comment (tisfeng) [inline Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift:24]: Wrong default values, it's just `option + a, s, d, f` one-line key order.

image

**Review by tisfeng — CHANGES_REQUESTED** [review]

**Review by AkaShark — APPROVED** [review]

**Review by Jerry23011 — APPROVED** [review]

**Review by phlpsong — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/cn_bug_report_zh.yml [hunk 3]
```diff
@@ -9,8 +9,6 @@ body:
     attributes:
       label: 请先确认以下事项：
       options:
-        - label: 请务必查看 [常见问题](https://github.com/tisfeng/Easydict/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98)
```
Comment (tisfeng) [inline .github/ISSUE_TEMPLATE/cn_bug_report_zh.yml:12]: 没注意，这个 revert 怎么把 issue 模版都改了啊？

Comment (AkaShark) [reply .github/ISSUE_TEMPLATE/cn_bug_report_zh.yml:12]: 😂

Comment (tisfeng) [reply .github/ISSUE_TEMPLATE/cn_bug_report_zh.yml:12]: 我刚 revert 改回来了 c9761e02

**Pull request merged**